pybikes/data/cocacola.json

```diff
@@ -0,0 +1,33 @@
+{
+    "instances": [
+        {
+            "tag": "cork",
+            "meta": {
+                "latitude": 51.893604,
+                "longitude": -8.494174,
+                "city": "Cork",
+                "country": "IE"
+            }
+        },
+        {
+            "tag": "limerick",
+            "meta": {
+                "latitude": 52.668746,
+                "longitude": -8.627809,
+                "city": "Limerick",
+                "country": "IE"
+            }
+        },
+        {
+            "tag": "galway",
```

Comment:
I will set it as system_id, as on bicincitta. It would be nice if we could "standardize" the name of those attributes too.